### PR TITLE
[fix bug 1351424] Enable Firefox Screenshots

### PR DESCRIPTION
**Pull request by ckprice**
Used this as a template https://github.com/mozilla/one-off-system-add-ons/tree/master/addons/hsts-priming

Fixes https://bugzilla.mozilla.org/show_bug.cgi?id=1351424 (note: this pref hasn't quite landed in Nightly just yet)

r? @felipc

### CODE REVIEW

File: addons/enable-screenshots/bootstrap.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ /* exported install, uninstall, startup, shutdown */
+ /* eslint no-implicit-globals: "off" */
+
+const {classes: Cc, interfaces: Ci, utils: Cu} = Components;
+
+const {Preferences} = Cu.import("resource://gre/modules/Preferences.jsm", this);
```
Comment (Standard8) [inline addons/enable-screenshots/bootstrap.js]: Please can you change `this` to `{}`

**Comment by Standard8** [comment]
@ckprice Also, please update to latest master to check it passes the latest ESLint changes. Thanks!

**Comment by ckprice** [comment]
Updated to latest master, and now passes checks. Couldn't find a cool way to squash these down to 1 commit.

**Comment by ckprice** [comment]
Going to go ahead and merge this since it was r+wc and those have been done.

**Pull request merged**